tests/foreman/api/test_organization.py

```diff
@@ -304,3 +305,85 @@ def test_negative_update(self, attrs):
         )
         with self.assertRaises(HTTPError):
             response.raise_for_status()
+
+    @skip_if_bug_open('bugzilla', 1103157)
+    def test_bugzilla_1103157(self):
+        """@Test: Create organization and add two compute resources
+        one by one using different transactions and different users
+        to see that they actually added, but not overwrite each other
+
+        @Feature: Organization - Update
+
+        @Steps:
+
+        1. Use the admin user to create an organization and two compute
+           resources. Make one compute resource point at / belong to the
+           organization.
+        2. Create a user and give them the ability to update compute resources
+           and organizations. Have this user make the second compute resource
+           point at / belong to the organization.
+        3. Use the admin user to read information about the organization.
+           Verify that both compute resources are pointing at / belong to the
+           organization.
+
+        @Assert: Organization contains both compute resources
+
+        """
+        compute_resource_ids = [
+            entities.ComputeResource(
+                name=gen_string('alpha'),
+                provider='Libvirt',
+                url='qemu://host.example.com/system'
+                ).create_json()['id']
+            for _
+            in range(2)
+        ]
+
+        client.put(
+            self.organization.path(),
+            {'organization': {
```

Comment:
Is this the right direction to move? It seems pretty weird to me:

```python
client.put(
    self.organization.path(), {
        'organization': {
            'compute_resource_ids': [compute_resource_ids[1]]
        }
    },
    verify=False,
    auth=(login, password),
).raise_for_status()
```

I think placing each argument by itself is a pretty straightforward way to write things, and placing `{` or `}` on the same line as another argument violates that and makes things harder to read.